cmd/up.go

```diff
@@ -83,13 +84,13 @@ func runCmdUp(cmd *cobra.Command, args []string) error {
 		if err := ioutil.WriteFile(templatePath, stackTemplate, 0600); err != nil {
 			return fmt.Errorf("Error writing %s : %v", templatePath, err)
 		}
-		if conf.KMSKeyARN == "" {
+		if conf.KMSKeyARN != "" {
 			fmt.Printf("BEWARE: %s contains your TLS secrets!\n", templatePath)
```

Comment:
Btw do you know what is this message meant for?
AFAIK, stack templates doesn't contain raw TLS secrets but KMS-encrypted versions of them.
If so, why do we need a warning(?) like this?